Fix bad code

ar1ja opened the pull request.

ar1ja committed:
Fix bad code

RaptaG commented:
This code is too advanced for me to get it (lol) I'm more if a beginner bash scripter and this is a very simple script, just a combination of the provided commands in the chaotic aur website. I'm not sure if I'll merge it..

RaptaG committed:
Improve and cleanup code

- More human-readable names
- Separate commands from the “main” command
- General improvements

RaptaG committed:
`”` —> `’` and other fixes

RaptaG commented:
Ok so I made some changes to clean your script up, but I need you approval @TruncatedDinosour, since you are better than me on coding 😂

RaptaG committed:
Remove useless line

RaptaG committed:
Oops

RaptaG committed:
Damn

RaptaG committed:
Fix version
